test_runner/src/main/kotlin/ftl/reports/outcome/BillableMinutes.kt

```diff
@@ -0,0 +1,46 @@
+package ftl.reports.outcome
+
+import com.google.api.services.toolresults.model.Step
+import com.google.api.services.toolresults.model.StepDimensionValueEntry
+import ftl.android.AndroidCatalog
+import ftl.util.billableMinutes
+import kotlin.math.min
+
+data class BillableMinutes(
+    val virtual: Long = 0,
+    val physical: Long = 0
+)
+
+fun List<Step>.calculateAndroidBillableMinutes(
+    projectId: String,
+    timeoutValue: Long
+): BillableMinutes =
```

Comment:
` : BillableMinutes ` could be skipped

Reply:
Sometimes, I prefer to leave the return type explicitly. Especially for public functions, where constructor call is not a first.